pkg/storage/stores.go

```diff
@@ -41,6 +41,7 @@ type ClusterStore interface {
 
 type RBACStore interface {
 	CreateRole(context.Context, *corev1.Role) error
+	UpdateRole(context.Context, *corev1.Role) error
```

Comment:
Can you add tests for this method to [RBACStoreTestSuite](https://github.com/rancher/opni/blob/24d21f0427ca7e41ea639fa18798f407f738e68c/pkg/test/conformance/storage/rbac_store.go#L15)?